test/hydrators.js

```diff
@@ -6,7 +6,11 @@ const appTester = zapier.createAppTester(App);
 describe('hydrators', () => {
 
   describe('uploadFile', () => {
-    it('should download files', (done) => {
+    it('should download files', function(done) {
+      if (!process.env.ZAPIER_DEPLOY_KEY || process.env.ZAPIER_DEPLOY_KEY === 'undefined') {
```

Comment:
Interestingly, we'll set this variable with the string `undefined` if we don't find a `~/.zapierrc` file.

Reply:
Something we can fix instead of workaround?

Reply:
I don't think it's a priority, so won't fix it now. Probably later (and this code should still work).